src/components/ProcessCreate/Steps/Confirm.tsx

```diff
@@ -242,14 +243,15 @@ export const Confirm = () => {
         >
           {t('form.process_create.confirm.create_button')}
         </Button>
-        <Modal isOpen={isOpen} onClose={onClose} closeOnEsc={!!error} closeOnOverlayClick={!!error} isCentered>
+        <Modal isOpen={true} onClose={onClose} closeOnEsc={!!error} closeOnOverlayClick={!!error} isCentered>
```

Comment:
@gerouvi OMFG I don't know how did I miss this.... but this change basically broke the creation of processes. You have to pay more attention please (me too obviously, since this should not have passed my review).